===================================================================
RCS file: /cvsroot/tora/tora/rpm/tora.sh,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- rpm/tora.sh	2003/07/22 20:14:15	1.1
+++ rpm/tora.sh	2005/03/29 19:15:47	1.2
@@ -2,9 +2,9 @@
 
 if [ "X$ORACLE_HOME" = "X" ]
 then
-    echo "Environment variable ORACLE_HOME not set, check your Oracle installation"
+    echo "WARNING: Environment variable ORACLE_HOME not set, check your Oracle installation"
 else
     LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ORACLE_HOME/lib
     export LD_LIBRARY_PATH
-    tora.real
 fi
+tora.real
